From 29c278215529ae0f48cb5ad10ba6414e3dd9afe8 Mon Sep 17 00:00:00 2001
From: Robert Richter <rrichter@cavium.com>
Date: Thu, 25 Jun 2015 16:59:41 +0200
Subject: [PATCH 129/131] acpi, property: Fix EXPORT_SYMBOL_GPL() for
 acpi_dev_prop_read_single()
Git-commit: 8199d3f72eec68dfdbdf0e5aa4acadf6e6a88a39
Patch-mainline: Queued in subsystem maintainer repository
Git-repo: git://git.kernel.org/pub/scm/linux/kernel/git/rric/linux.git thunder/master-v4.1

Fixing the following build error when building drivers as modules:

 ERROR: "acpi_dev_prop_read_single" [drivers/net/phy/mdio-octeon.ko] undefined!
 ERROR: "acpi_dev_prop_read_single" [drivers/net/ethernet/cavium/thunder/thunder_bgx.ko] undefined!

Reported-by: Andreas Schwab <schwab@suse.de>
Signed-off-by: Robert Richter <rrichter@cavium.com>

Signed-off-by: Matthias Brugger <mbrugger@suse.com>

---
 drivers/acpi/property.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/acpi/property.c b/drivers/acpi/property.c
index 22d66fb..a1d96c4 100644
--- a/drivers/acpi/property.c
+++ b/drivers/acpi/property.c
@@ -424,7 +424,7 @@ int acpi_dev_prop_read_single(struct acpi_device *adev, const char *propname,
 	}
 	return ret;
 }
-EXPORT_SYMBOL_GPL(acpi_dev_prop_read);
+EXPORT_SYMBOL_GPL(acpi_dev_prop_read_single);
 
 static int acpi_copy_property_array_u8(const union acpi_object *items, u8 *val,
 				       size_t nval)
-- 
1.7.12.4

